text/fourth-breaking.md

```diff
@@ -191,46 +172,12 @@ PLEASE.
 ```text
 cargo build
    Compiling lists v0.1.0 (file:///Users/ABeingessner/dev/too-many-lists/lists)
-src/fourth.rs:58:13: 58:27 error: use of unstable library feature 'rc_unique'
-src/fourth.rs:58             Rc::try_unwrap(old_head).ok().unwrap().into_inner().elem
-                             ^~~~~~~~~~~~~~
-note: in expansion of closure expansion
-src/fourth.rs:48:30: 59:10 note: expansion site
-src/fourth.rs:58:13: 58:27 help: add #![feature(rc_unique)] to the crate attributes to enable
-error: aborting due to previous error
-Could not compile `lists`.
-```
-
-NO.
-
-> add `#![feature(rc_unique)]` to the crate attributes to enable
-
-FINE.
-
-```rust
-// in lib.rs
-#![feature(rc_unique)]
-
-pub mod first;
-pub mod second;
-pub mod third;
-pub mod fourth;
-pub mod fifth;
-```
-
-```text
-cargo build
-   Compiling lists v0.1.0 (file:///Users/ABeingessner/dev/too-many-lists/lists)
 ```
 
 YES.
 
 *phew*
 
-Ok what was that last thing? Rust *really* doesn't want you using unstable
-features, so it makes you explicitly name every single one you do at the top
-of your crate. *shrug*
-
 As one of the people who makes these decisions, I can just say that `try_unwrap`
```

Comment:
Please also remove this sentence, as it refers to the time before `try_unwrap` was stabilized.